ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -1502,7 +1536,7 @@ const triggerSet: TriggerSet<Data> = {
           fr: 'LB TANK !!',
           ja: 'タンクLB!!',
           cn: '坦克LB！！',
-          ko: '리미트 브레이크!!',
+          ko: '탱리밋!!',
```

Comment:
This is what is used elsewhere for tank LB.  If you want to change this, you might want to fix the other uses of this too.

Reply:
I'll change other TANK LB triggers